flight/PiOS/Common/pios_mpu.c

```diff
@@ -0,0 +1,1090 @@
+/**
+ ******************************************************************************
+ * @file       pios_mpu.c
```

Comment:
I wonder if it would be better to rename this to "mpuxxxx". MPU for me has meant [other stuff](https://en.wikipedia.org/wiki/MPU), some of which could feasibly be on an autopilot.